plugins/kotlin/fir/src/org/jetbrains/kotlin/idea/completion/contributors/FirCallableCompletionContributor.kt

```diff
@@ -125,13 +140,15 @@ internal open class FirCallableCompletionContributor(
         when {
             symbol is KtPackageSymbol -> collectDotCompletionForPackageReceiver(symbol, expectedType, visibilityChecker)
             symbol is KtNamedClassOrObjectSymbol && symbol.classKind == KtClassKind.ENUM_CLASS -> {
-                collectNonExtensions(symbol.getStaticMemberScope(), visibilityChecker, scopeNameFilter).forEach { memberSymbol ->
-                    addCallableSymbolToCompletion(
-                        expectedType,
-                        memberSymbol,
-                        CallableInsertionOptions(ImportStrategy.DoNothing, getInsertionStrategy(memberSymbol))
-                    )
-                }
+                collectNonExtensions(symbol.getStaticMemberScope(), visibilityChecker, scopeNameFilter)
+                    .forEach { memberSymbol ->
+                        addCallableSymbolToCompletion(
+                            expectedType,
+                            memberSymbol,
+                            CallableInsertionOptions(ImportStrategy.DoNothing, getInsertionStrategy(memberSymbol) ?: return@forEach),
```

Comment:
Please, do not use inline return/break/continue statements. Introduce a variable for that